sql3/sql_test.go

```diff
@@ -95,6 +101,32 @@ func TestSQL_Execute(t *testing.T) {
 								for _, row := range rows {
 									assert.Contains(t, exp, row)
 								}
+							case defs.ComparePartial:
```

Comment:
What do we need this for?

Reply:
We have tests of `SHOW TABLES` and `SHOW COLUMNS`. The results from those include timestamps of when things were created, making it impossible for statically-defined test data to match them. The best solution I could think of was to add support for "we know the following about the results"; for instance, "we should have a field named `age` with type `int`", but we can't actually write the full row of the response.

This might apply in other cases where the full results from a query aren't predictable, but we can say things about what has to be there. This is similar to the existing `CompareIncludedIn`, but that is testing that full rows are somewhere in the output results, and we can't actually provide full rows.

Reply:
Got it.